.github/workflows/release.yml

```diff
@@ -0,0 +1,13 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    uses: guardian/gha-scala-library-release-workflow/.github/workflows/reusable-release.yml@main
+    permissions:
+      contents: write
```

Comment:
This is missing one permission - see the example [`release.yml`](https://github.com/guardian/etag-caching/blob/main/.github/workflows/release.yml) referenced by the docs at [configuration.md#github-workflow](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/docs/configuration.md#github-workflow) 

_(https://github.com/guardian/gha-scala-library-release-workflow/pull/19 introduced an extra required permission)_